html-forms/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email-address">Email</label>
+      <div>
+        <input name="email" type="email" id="email-address">
+      </div>
+      <label for="why-coding">Why is coding cool?</label>
+      <div>
+        <select name="coding">
+          <option value="">Select an option</option>
+          <option value="interesting">Because it&apos;s interesting</option>
+          <option value="fun">Because it&apos;s fun</option>
+          <option value="powerful">Because it&apos;s powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <label>
+        <input type="radio" name="language" value="html">HTML
+        <input type="radio" name="language" value="css">CSS
+        <input type="radio" name="language" value="javascript">Javascript
+      </label>
```

Comment:
There should be an individual label for each `input`, not one surrounding all of them